### PR TITLE
Pull token data from IndieAuth plugin

### PR DESCRIPTION
**Pull request by dshanske**
I noticed that when I use the IndieAuth endpoint, the auth response storage vanished. This restores that behavior.

### CODE REVIEW

File: micropub.php [hunk 1]
```diff
@@ -175,6 +174,10 @@ public static function parse_query( $wp ) {
 			if ( ! empty( $indieauth_scopes ) ) {
 				static::$scopes = $indieauth_scopes;
 			}
+			global $indieauth_token;
```
Comment (snarfed) [inline micropub.php]: huh. is this really an ok way for plugins to communicate?! seems like they should use hooks or filters instead?

Comment (snarfed) [reply micropub.php]: i do get that the code was already working this way, and this just extends that. still though.

Comment (dshanske) [reply micropub.php]: I am happy to see explore changibg it to a filter on both ends

Comment (snarfed) [reply micropub.php]: thanks! if you plan to switch to filters, feel free to merge this now and do that afterward.

**Comment by dshanske** [comment]
@snarfed Switches to filters.

**Pull request merged**